tests/Feature/OptimizedFilepondServiceTest.php

```
@@ -0,0 +1,137 @@
+function createService(): OptimizedFilepondService
+{
+    $service = new OptimizedFilepondService(
+        app(UploaderManager::class)
+    );
+
+    return $service;
+}
+
+function processImageFile(OptimizedFilepondService $service, UploadedFile $file, Request $request): UploadedFile
+{
+    $method = new ReflectionMethod($service, 'processImageFile');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '21', column '19'). (undefined)

(MissingImport)